permissions/plugin-insightappsec.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "insightappsec"
+github: "jenkinsci/insightappsec-plugin"
+paths:
+- "org/jenkins-ci/plugins/insightappsec"
```

Comment:
Side note: we tend to recommend people to now prefer `io.jenkins.plugins`, but not a requirement.